creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -131,8 +131,22 @@ const healthCheckController = async (req) => {
  * Controller for `health_check/sync` route, calls
  * syncHealthCheckController
  */
-const syncHealthCheckController = async () => {
+const syncHealthCheckController = async (req) => {
```

Comment:
i just used this as an example of a Gauge metric - the previous example in Monitors was actually broken